src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -796,6 +802,18 @@ internal void SetRedoButtonStatus(bool enabled)
         redoButton.Disabled = !enabled;
     }
 
+    internal void SetError(string message)
```

Comment:
I don't think we should have these general error things. There should instead be just a method to show the error about disconnected hexes popup.